lapis-e2e/test/alignedNucleotideSequence.spec.ts

```diff
@@ -170,33 +171,34 @@ describe('The /alignedNucleotideSequence endpoint', () => {
       });
     });
 
-    it('should ignore the fasta header template when returning JSON', async () => {
-      const result = await lapisMultiSegmentedSequenceController.postAlignedNucleotideSequence({
-        nucleotideSequenceRequest: {
-          dataFormat: 'JSON',
-          fastaHeaderTemplate: '{primaryKey}{date}{something invalid}',
-        },
-        segment: 'M',
+    it('should throw an error for fasta header template when returning JSON', async () => {
```

Comment:
I guess this function is now a duplication of the one above so we can remove it :-) (surprised there isn't a warning that two functions have the same name)

Reply:
Ah sorry I just saw this is for the multi-segmented case so you can ignore me